src/views/shared/exit/exit.view.jsx

```diff
@@ -286,18 +288,22 @@ function Exit({
           const pendingDelayedWithdrawal = pendingDelayedWithdraws?.find(
             (pendingDelayedWithdrawal) => pendingDelayedWithdrawal.id === exitId
           );
+          const availableWithdraw = availableWithdraws?.find(
+            (availableWithdraws) => availableWithdraws.id === exitId
+          );
 
-          if (pendingDelayedWithdrawal) {
-            const remainingTime = getDateString(pendingDelayedWithdrawal);
+          if (pendingDelayedWithdrawal || availableWithdraw) {
+            const withdraw = pendingDelayedWithdrawal || availableWithdraw;
+            const remainingTime = getDateString(withdraw, availableWithdraw);
             return (
               <div className={classes.withdraw}>
                 <div className={`${classes.withdrawInfo} ${classes.withdrawInfoDelayed}`}>
-                  {pendingDelayedWithdrawal.isInstant && (
+                  {withdraw && (
```

Comment:
This could be true if there is a pendingDelayedWithdrawal that is not instant. Might be worth checking availableWithdraw instead?

Reply:
Yes that logic doesn't have sense there